src/style.css

```
@@ -1,3 +1,72 @@
+/* stylelint-disable rule-empty-line-before */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
   margin: 0;
 }
+
+.link {
+  text-decoration: none;
+  color: #000;
```

don't forget to create variables for all colors
```suggestion
  color: $c-black;
```